README.md

```diff
@@ -1,195 +1,19 @@
+from any release before 0.5.1 to 0.5.1 or later, this script must be applied to your Registry's Opensearch endpoint.**
 
-## 🏃 Getting Started With This Template
+The syntax for invoking this script is as follows:
 
-See our wiki page for more info on setting up your new repo. You can remove this section once you have completed the necessary start-up steps.
+```archive-status.sh <Opensearch URL> [<Opensearch username> <Opensearch password>]```
 
-https://github.com/NASA-PDS/nasa-pds.github.io/wiki/Git-and-Github-Guide#creating-a-new-repo
-
-**👉 Important!** You must assign the teams as mentioned on the wiki page above! At a minimum, these are:
-
-| Team                                | Permission |
-| ----------------------------------- | ---------- |
-| `@NASA-PDS/pds-software-committers` | `write`    |
-| `@NASA-PDS/pds-software-pmc`        | `admin`    |
-| `@NASA-PDS/pds-operations`          | `admin`    |
-
----
-
-# My Project
-
-This is the XYZ that does this, that, and the other thing for the Planetary Data System.
-
-Please visit our website at: https://nasa-pds.github.io/pds-my-project
-
-It has useful information for developers and end-users.
-
--   [📀 Quick Start: Installation and Usage](#user-content--quick-start-installation-and-usage)
-    -   [💁‍♀️ Using this Package](#user-content-️-using-this-package)
--   [👥 Contributing](#user-content--contributing)
-    - [🔢 Versioning](#user-content--versioning)
-    - [🪛 Development](#user-content--development)
-        -   [🚅 Continuous Integration & Deployment](#user-content--continuous-integration--deployment)
-        -   [🔧 Manual Publication](#user-content--manual-publication)
-            -   [Update Version Numbers](#user-content-update-version-numbers)
-            -   [Update Changelog](#user-content-update-changelog)
-            -   [Commit Changes](#user-content-commit-changes)
-            -   [Build and Deploy Software to Maven Central Repo](#user-content-build-and-deploy-software-to-maven-central-repo)
-            -   [Push Tagged Release](#user-content-push-tagged-release)
-            -   [Deploy Site to Github Pages](#user-content-deploy-site-to-github-pages)
-            -   [Update Versions For Development](#user-content-update-versions-for-development)
-            -   [Complete Release in Github](#user-content-complete-release-in-github)
--   [📃 License](#user-content--license)
-
-
-## 📀 Quick Start: Installation and Usage
-
-_Installation instructions here. Include any system-wide requirements (`bres install`, `apk add`, `apt-get intstall`, `yum install`, "CMMI" (configure, make, make install), etc.)._
-
-
-### 💁‍♀️ Using this Package
-
-_Basic usage instructions here. If possible, make it so your program works correctly "out of the box" as an executable `.jar`, or as a drop-in `.war`, etc., without any additional configuration._
-
-
-## 👥 Contributing
```

Comment:
Might be nice to keep the contributor guidelines to show our dedication to ethical behavior